juju/client/_client11.py

```diff
@@ -3697,27 +3692,6 @@ async def SetSupportedContainers(self, params=None):
 
 
 
-    @ReturnMapping(StringsResult)
-    async def StateAddresses(self):
```

Comment:
This is a bit scary...

Reply:
Why is that?

Reply:
As we shouldn't be removing methods from the API server i.e. it's always additive. 

I believe this is ok, as it's a change @wallyworld did https://github.com/juju/juju/pull/12184